src/express.ts

```diff
@@ -195,7 +196,10 @@ export class InngestCommHandler {
           "Unable to determine your site URL to serve the Inngest handler.";
         console.error(message);
 
-        return res.status(500).json({ message });
+        return res
+          .status(500)
+          .set("x-inngest-sdk", `js/${this.frameworkName}`)
```

Comment:
Can we refactor the `js/${this.frameworkName}` to use the same as the `User-Agent` header? See [src/express.ts#299](https://github.com/inngest/inngest-js/blob/521a4d296758f5ea5560ee0c7679fe5c101ac3e0/src/express.ts#L299)

Abstracting that and this out to `this.sdkHeader()` might be nice?